include/zephyr/posix/signal.h

```diff
@@ -92,12 +102,34 @@ struct sigevent {
 	int sigev_signo;
 };
 
+typedef struct {
+	int si_signo;
+	int si_code;
+	union sigval si_value;
+} siginfo_t;
+
+struct sigaction {
+	void (*sa_handler)(int signno);
+	sigset_t sa_mask;
+	int sa_flags;
+	void (*sa_sigaction)(int signo, siginfo_t *info, void *context);
+};
```

Comment:
Shouldn't it be
```
struct sigaction {
	void (*sa_handler)(int signno);
	void (*sa_sigaction)(int signo, siginfo_t *info, void *context);
	sigset_t sa_mask;
	int sa_flags;
};
```
to avoid possible hole in the struct.

Reply:
Fixed!